google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineStandardFacet.java

```diff
@@ -29,23 +29,23 @@
 /**
  * @author nik
  */
-public class AppEngineFacet extends Facet<AppEngineFacetConfiguration> {
+public class AppEngineStandardFacet extends Facet<AppEngineFacetConfiguration> {
 
-  public static final FacetTypeId<AppEngineFacet> ID = new FacetTypeId<>("appEngine");
+  public static final FacetTypeId<AppEngineStandardFacet> ID = new FacetTypeId<>("appEngine");
```

Comment:
so I guess the fact that 'appEngine' doesn't change means all existing serialized run configs keeps on working?